### ISSUE TITLE
Tooltip - persistent on hover

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1457]
To meet AA WCAG 2.1 compliance, Tooltips should be persistent on hover (i.e. when you move the mouse from the tooltip toggle to the tooltip contents.

The WCAG item with more details about this: [WCAG 1.4.13](https://www.w3.org/WAI/WCAG21/quickref/#content-on-hover-or-focus)

**Comment by ibolton336** [comment]
+1 Would like to see this functionality added.
